### ISSUE TITLE
unit tests

### ISSUE THREAD

**Issue by cpadavis** [cpadavis/billy#33]
@ajmcleod and I have written several little scripts along the way to make sure our code works, but we should formalize these into unit tests, especially since our older tests are broken due to api changes!

There are two main "branches" that should be dealt with:

1. [ ] Creation of reasonable "miniature" catalogs (say, 4 entries from each dataset) that we can store with the repo.
2. [ ] Creation of the unit tests. In particular, we should always be able to:
    - [ ] construct each dataset
    - [ ] run the training, with ability to calculate training and test accuracies
    - [ ] save and load conv nets
    - [ ] fix a layer and ensure it doesn't change in resulting training
    - [ ] transfer learn

**Comment by drphilmarshall** [comment]
Good idea. You should set up Travis CI (see the repo settings "webhooks and
services") to run nosetests on the repo once you have written your first
test.
